config.go

```diff
@@ -9,11 +9,35 @@ import (
 
 // Configuration is a minimal set of parameters for Sentry integration.
 type Configuration struct {
-	Tags              map[string]string
+	// Tags are passed as is to the corresponding sentry.Event field.
+	Tags map[string]string
+
+	// LoggerNameKey is the key for zap logger name.
+	// If not empty, the name is added to the rest of zapcore.Field(s),
+	// so that be careful with key duplicates.
+	// Leave LoggerNameKey empty to disable the feature.
+	LoggerNameKey string
+
+	// DisableStacktrace disables adding stacktrace to sentry.Event, if set.
 	DisableStacktrace bool
-	Level             zapcore.Level
-	BreadcrumbLevel   zapcore.Level
-	EnableBreadcrumbs bool // this feature works only when you explicitly passed new scope
-	FlushTimeout      time.Duration
-	Hub               *sentry.Hub
+
+	// Level is the minimal level of sentry.Event(s).
+	Level zapcore.Level
+
+	// BreadcrumbLevel is the minimal level of sentry.Breadcrumb(s).
+	// Breadcrumb specifies an application event that occurred before a Sentry event.
```

Comment:
I would write something like this, to make it more understandable (please, counter this with something less verbose :D its just an idea/direction I would take):
`
Breadcrumb is an event that was logged but not sent to Sentry prior the current event and is used to enhance the currently processed event (for example, an error event would include breadcrumbs logged as debug, even though Configuration.Level specifies higher level like warning or error).
`

Reply:
I'm not familiar with breadcrumbs and have never used them.
I just copied a part of go-sentry Breadcrumb struct docs.
That's it.

Reply:
Yeah, maybe it's enough to refer to the Sentry docs. 👍